app/views/api/services/edit.html.slim

```diff
@@ -1,4 +1,4 @@
-- content_for :sublayout_title, 'Naming'
+- content_for :page_header_title, 'Naming'
```

Comment:
Wow, WTF, this is an **extremely** confusing title :flushed: 

It seems that it has always been like that, but I'm kinda shocked :sweat_smile: 

Should be "Edit Product" to me...

Reply:
I stopped getting porta-shocked a long time ago 🙂.